airflow-core/tests/unit/utils/test_preexisting_python_virtualenv_decorator.py

```diff
@@ -21,7 +21,7 @@
 
 import pytest
 
-from airflow.utils.decorators import remove_task_decorator
+from airflow.sdk.bases.decorator import remove_task_decorator
```

Comment:
We shouldn't be importing from sdk here, created a dependency which we are trying to chop.

This function could probably be easily duplicated.